pkg/core/dao/dao.go

```diff
@@ -556,7 +555,7 @@ func (dao *Simple) StoreAsBlock(block *block.Block, buf *io.BufBinWriter) error
 }
 
 // DeleteBlock removes block from dao.
-func (dao *Simple) DeleteBlock(h util.Uint256, w *io.BufBinWriter) error {
+func (dao *Simple) DeleteBlock(h util.Uint256, p2pSigExtensions bool, w *io.BufBinWriter) error {
```

Comment:
We can hide `p2pSigExtensions` in `Simple` itself, `NewSimple` can take it as an argument.